# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ShareableCanvasRenderer.cpp
# Commit: cdaa8a4e9e36
# Full Hash: cdaa8a4e9e366ad041b429a8f5bd0b9b259d4722
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - NON_PREMULT TextureClient iff NON_PREMULT CanvasClient. r=handyman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78800
# ==============================================================================

diff -r 9ff26bcc580c -r cdaa8a4e9e36 gfx/layers/ShareableCanvasRenderer.cpp
--- a/gfx/layers/ShareableCanvasRenderer.cpp	Mon Jun 08 20:35:38 2020 +0000
+++ b/gfx/layers/ShareableCanvasRenderer.cpp	Mon Jun 08 20:58:03 2020 +0000
@@ -159,7 +159,8 @@
 
       const RefPtr<DrawTarget> dt = tc->BorrowDrawTarget();
 
-      const auto borrowed = BorrowSnapshot();
+      const bool requireAlphaPremult = false;
+      const auto borrowed = BorrowSnapshot(requireAlphaPremult);
       if (!borrowed) return nullptr;
 
       dt->CopySurface(borrowed->mSurf, {{0, 0}, size}, {0, 0});